src/dashboard/Data/Browser/ExportSelectedRowsDialog.react.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2016-present, Parse, LLC
+ * All rights reserved.
+ *
+ * This source code is licensed under the license found in the LICENSE file in
+ * the root directory of this source tree.
+ */
+import Modal     from 'components/Modal/Modal.react';
+import React     from 'react';
+
+export default class ExportSelectedRowsDialog extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      confirmation: ''
+    };
+  }
+
+  valid() {
+    return true;
+  }
+
+  render() {
+    let selectionLength = Object.keys(this.props.selection).length;
+    return (
+      <Modal
+        type={Modal.Types.INFO}
+        icon='warn-outline'
+        title={this.props.selection['*'] ? 'Export all rows?' : (selectionLength === 1 ? 'Export this row?' : `Export ${selectionLength} rows?`)}
+        subtitle={''}
+        disabled={!this.valid()}
+        confirmText={'Yes export'}
+        cancelText={'Never mind, don\u2019t.'}
+        onCancel={this.props.onCancel}
+        onConfirm={this.props.onConfirm}>
+        {}
+      </Modal>
```

Comment:
Can we add to this modal an info about 10,000 rows limit?

Reply:
Sure, let me see how I can work that in.

Reply:
The one you posted in the main thread looks perfect 👏👏👏👏

Reply:
Sorry I didn't see your response until just now. Oops!

I'll commit this now.